crates/ruma-events/src/room/message.rs

```diff
@@ -958,6 +958,7 @@ fn get_plain_quote_fallback(original_message: &MessageEvent) -> String {
     }
 }
 
+#[allow(clippy::nonstandard_macro_braces)]
```

Comment:
This still has a few problems unfortunatly the `format!` is triggered because `formatdoc!` contains "format". I wonder if I should have done `macro_name_to_check.starts_with(&format("{}!", name)`

Reply:
While I agree that's a bug, I also think we should use regular parentheses for `formatdoc!` (as we might already do in some places).

Reply:
Do you want me to add it to the list, I don't see anywhere where we don't use `formatdoc!(..)` but the point is catching errors in the future as much as clean up...

Reply:
> Do you want me to add it to the list

Sure.

> I don't see anywhere where we don't use `formatdoc!(..)`

Then why did the lint fire?

Reply:
Because `formatdoc!` starts with `format` I opened another clippy fix PR https://github.com/rust-lang/rust-clippy/pull/7478/commits/c1786d260cb7e20cfba6be9804748cc0bb0c9d60